sdl-core/src/main/scala/io/smartdatalake/workflow/action/spark/transformer/ScalaClassSparkDs2To1Transformer.scala

```diff
@@ -57,7 +57,7 @@ case class ScalaClassSparkDs2To1Transformer(override val name: String = "ScalaCl
 
     options("parameterResolution") match {
       case "dataObjectId" => Map(outputDo.id.id -> customTransformer.transformBasedOnDataObjectId(context.sparkSession, options, dfs))
-      case "dataObjectOrdering" => Map(outputDo.id.id -> customTransformer.transformBasedOnDataObjectOrder(context.sparkSession, options, inputDos, dfs))
+      //case "dataObjectOrdering" => Map(outputDo.id.id -> customTransformer.transformBasedOnDataObjectOrder(context.sparkSession, options, inputDos, dfs))
```

Comment:
I would still like to keep this as an option. I've seen in my project the need to re use the same DatasetTransformation in different places. I think it should also be possible with your implementation. Do you mind if I re-implement it for your new transformer?

Reply:
Yes sure. I just made a draft how we could implement Nto1.